packages/forms/test/form_group_spec.ts

```diff
@@ -1825,6 +1825,340 @@ describe('FormGroup', () => {
     });
   });
 
+  describe('emit `statusChanges` and `valueChanges` with/without async/sync validators', () => {
+    describe('stand alone controls', () => {
+      it('should run the async validator on stand alone controls and set status to `INVALID`',
+         fakeAsync(() => {
+           const logger: string[] = [];
+           const c =
+               new FormControl('', null, simpleAsyncValidator({timeout: 0, shouldFail: true}));
+           c.valueChanges.subscribe((value) => logger.push(`value: ${value}`));
+           c.statusChanges.subscribe((status) => logger.push(`status: ${status}`));
+
+           expect(logger.length).toBe(0);
+
+           tick(1);
+
+           c.setValue('new!', {emitEvent: true});
+
+           tick(1);
+
+           expect(logger).toEqual([
+             'status: INVALID',  // status change emitted as a result of initial async validator run
+             'value: new!',      // value change emitted by `setValue`
+             'status: INVALID'   // async validator run after `setValue` call
+           ]);
+         }));
+
+      it('should run the async validator on stand alone controls and set status to `VALID`',
+         fakeAsync(() => {
+           const logger: string[] = [];
+           const c = new FormControl('', null, asyncValidator('new!'));
+           c.valueChanges.subscribe((value) => logger.push(`value: ${value}`));
+           c.statusChanges.subscribe((status) => logger.push(`status: ${status}`));
+
+           expect(logger.length).toBe(0);
+
+           tick(1);
+
+           c.setValue('new!', {emitEvent: true});
+
+           tick(1);
+
+           expect(logger).toEqual([
+             'status: INVALID',  // status change emitted as a result of initial async validator run
+             'value: new!',      // value change emitted by `setValue`
+             'status: PENDING',  // status change emitted by `setValue`
+             'status: VALID'     // async validator run after `setValue` call
+           ]);
+         }));
+
+      it('should run setValue before the initial async validator and set status to `VALID`',
+         fakeAsync(() => {
+           const logger: string[] = [];
+           const c = new FormControl('', null, asyncValidator('new!'));
+           c.valueChanges.subscribe((value) => logger.push(`value: ${value}`));
+           c.statusChanges.subscribe((status) => logger.push(`status: ${status}`));
+
+           expect(logger.length).toBe(0);
+
+           c.setValue('new!', {emitEvent: true});
+
+           tick(1);
+
+           expect(logger).toEqual([
```

Comment:
I'd propose to add a quick comment here:
```suggestion
           // The `setValue` call invoked synchronously cancels the initial run of the `asyncValidator` (which 
           // would cause the control status to be changed to `INVALID`), so the log contains only events after
           // calling `setValue`. 
           expect(logger).toEqual([
```